tests/DCPS/MultiTopic/MultiTopicTest.cpp

```diff
@@ -258,28 +609,22 @@ bool run_multitopic_test(const Publisher_var& pub, const Subscriber_var& sub)
   }
 
   // Dispose
-
-  ReturnCode_t ret = midw->dispose(mi, HANDLE_NIL);
-  if (ret != RETCODE_OK) return false;
+  check_rc(midw->dispose(mi, HANDLE_NIL), "dispose mi");
   ReadCondition_var rc =
     dr->create_readcondition(ANY_SAMPLE_STATE, ANY_VIEW_STATE,
                              NOT_ALIVE_DISPOSED_INSTANCE_STATE);
   WaitSet_var ws = new WaitSet;
   ws->attach_condition(rc);
-  const Duration_t infinite = {DURATION_INFINITE_SEC, DURATION_INFINITE_NSEC};
   ConditionSeq active;
-  ret = ws->wait(active, infinite);
-  if (ret != RETCODE_OK) return false;
+  check_rc(ws->wait(active, max_wait), "wait for dispose");
   ws->detach_condition(rc);
   ResultingDataReader_var res_dr = ResultingDataReader::_narrow(dr);
   ResultingSeq data;
   SampleInfoSeq info;
-  ret = res_dr->read_w_condition(data, info, LENGTH_UNLIMITED, rc);
+  check_rc(res_dr->read_w_condition(data, info, LENGTH_UNLIMITED, rc), "");
```

Comment:
Looks like this needs a description string

Reply:
Yes, can't imagine why I left this blank.